src/app/support/donationBox.tsx

```diff
@@ -330,12 +343,22 @@ function DonationBox() {
                 <>
                   <input
                     type="text"
-                    name="name"
-                    placeholder="Your Name*"
-                    value={formData.name}
+                    name="firstName"
+                    placeholder="Your First Name*"
+                    value={formData.firstName}
+                    onChange={handleInputChange}
+                    className="rounded-full border-2 border-jade bg-cream p-2 px-4 text-left text-sm placeholder-jade placeholder-opacity-50 drop-shadow-jade"
+                    aria-label="firstName"
+                    required
+                  />
+                  <input
+                    type="text"
+                    name="lastName"
+                    placeholder="Your Last Name*"
+                    value={formData.lastName}
                     onChange={handleInputChange}
                     className="rounded-full border-2 border-jade bg-cream p-2 px-4 text-left text-sm placeholder-jade placeholder-opacity-50 drop-shadow-jade"
-                    aria-label="name"
+                    aria-label="lastName"
                     required
                   />
                   <input
```

Comment:
It won't let me comment below for some reason... but where you declared 
`{formData.donationCategory === "business" &&
                isDonationCategory(formData.donationCategory) && ( 
                <>
                    {Object.entries(
                      supportPageInfo.rewards[formData.donationCategory],` you can just write it as:
                      `{formData.donationCategory === "business" && (
                  <>
                    {Object.entries(
                      supportPageInfo.rewards[formData.donationCategory as DonationCategory],`

Reply:
Ha, thanks for this! I honestly don't remember writing that... 🤣